server/github_api_calls.js

```diff
@@ -4,17 +4,17 @@ githubAuthRevoke = function(accessToken) {
   const clientSecret = Meteor.settings.github_clientsecret;
   const options = {
     headers: {
-      "User-Agent": "CodeBuddies",
-      "Authorization": `token ${accessToken}`
+      Accept: 'application/vnd.github.v3+json'
     },
-    auth: `${clientId}:${clientSecret}`
+    data: {
+      access_token: accessToken
+    }
   };
-  const response = HTTP.del(
-    "https://api.github.com/user/repos",
-    options
-  );
+  // please see the following documentation
+  // https://docs.github.com/en/rest/reference/apps#delete-an-app-authorization--code-samples
+  const response = HTTP.del('https://api.github.com/applications/${clientId}/grant', options);
```

Comment:
My bad for the former change here... btw restoring it back to:

![image](https://user-images.githubusercontent.com/4512699/142807872-c32314ba-61eb-4264-b5ee-5708fb4975f5.png)

... should also work, right?